### DISCUSSION TITLE
Testing the trained model or opening the A1111 colab do nothing

### DISCUSSION THREAD

**Discussion by i-huzaifa-arshad** [TheLastBen/fast-stable-diffusion#1265] [General]
I downloaded your latest colabs for A1111 and DreamBooth. I trained a model and when I tried to test it, the cell just run and gave no link. Similar is with A1111 notebook. The DreamBooth colab cell to open A1111 gives this warning then do nothing.

> fatal: not a git repository (or any parent up to mount point /content)
> Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).

![image](https://user-images.githubusercontent.com/76521534/210762168-576a84d4-ea10-4062-8ecc-a7b319b0faa8.png)

**Comment by TheLastBen** [comment]
the A1111 github repo was suspended, I updated the link in the latest commit, https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast-DreamBooth.ipynb

**Reply by i-huzaifa-arshad** [reply]
Oh. Who suspended it and why? Is this means no A1111 WebUI?

**Reply by TheLastBen** [reply]
it means a different webhost that's all : https://gitgud.io/AUTOMATIC1111/stable-diffusion-webui

**Reply by i-huzaifa-arshad** [reply]
Oh Thanks God.
